web/src/portico/signup.ts

```diff
@@ -342,3 +343,49 @@ $(() => {
         }
     });
 });
+
+const $password_elem: JQuery<HTMLInputElement> = $("input#id_password");
+const $new_password1_elem: JQuery<HTMLInputElement> = $("input#id_new_password1");
+
+if ($password_elem.length) {
+    $password_elem.on("input", () => {
+        check_overflow_password($password_elem);
+    });
+}
+
+if ($new_password1_elem.length) {
+    $new_password1_elem.on("input", () => {
+        check_overflow_password($new_password1_elem);
+    });
+}
+
+export function check_overflow_password($password_elem: JQuery<HTMLInputElement>): void {
+    const password = $password_elem.val() ?? "";
```

Comment:
I guess `?? ""` is to handle typescript linter. We usually do `val()!` in existing code, as we do not expect `val()` to return null or undefined for any case.